index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://github.com/bilalrajput09" class="name">
+        Bilal Ahmed
+      </a>
+        <img src="C:\Users\HP\My-Portfolio\img\threelines.png">
+    </div>  
+  </header>
+  <div class="introduction">
+    <h1>Hey There.
+      I’m Bilal Ahmed</h1>
+      <h2>I’am a Software Developer</h2>
+      <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</h3>
+  </div>
```

Comment:
- [ ] You did well👌, but I see that the margin of the elements is not like in the Figma Design, so kindly😊 try to add the correct margin like the Figma you can see the difference below⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212328612-18326f39-f98f-4cca-a12a-4337d69eae24.png) |   ![image](https://user-images.githubusercontent.com/83360636/212328814-eabeeed0-63a6-4bfd-aa9b-0e92ad25c64b.png) |